### PR TITLE
Package CloudTrail plus standardisation and Auto Remediation Playbooks

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: Internal

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by amshamah419** [comment]
@mayagoldb What changes (if any) were required for the playbooks?

**Comment by mayagoldb** [comment]
@yaron-libman please review the following files:
Playbooks/playbook-PCR_-_AWS_CloudTrail_Integrated_CloudWatchLogs.yml 
Playbooks/playbook-PCR_-_AWS_CloudTrail_Misconfig.yml 
Playbooks/playbook-PCR_-_AWS_CloudTrail_Trail_Misconfig.yml

**Comment by mayagoldb** [comment]
@amshamah419 I'm approving the playbooks

**Comment by amshamah419** [comment]
@mayagoldb Integration is passing the test playbook. I'll merge once we have a complete circle run.

File: Integrations/AWS-CloudTrail/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
+Minor bug fix/ spelling correction.
```
Comment (amshamah419) [inline Integrations/AWS-CloudTrail/CHANGELOG.md]:
```suggestion
- Bugfix for Proxy/Insecure issues.
- Fixed issue where spelling error was preventing some commands to execute.
```

**Pull request merged**